docs-gen/content/_index.md

```diff
@@ -42,3 +42,5 @@
 Uses the same great technologies like KFP itself.
 
 {{< /blocks/feature >}}
+
+{{< /blocks/section >}}
```

Comment:
Not having this closing section block now makes the build fail. It was just ignored previously